### PR TITLE
Update General: JSON

### PR DESCRIPTION
**Pull request by Dusch4593**
### Description

Update inspired from information from this [Codecademy article](https://www.codecademy.com/article/what-is-json) with similar content.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/json/json.md [hunk 1]
```diff
@@ -12,7 +12,17 @@ CatalogContent:
...
+<link rel="canonical" href="https://www.codecademy.com/article/what-is-json" />
+
+**JavaScript Object Notation (JSON)** is a data format that is readable, writable, and parsable for both humans and machines. JSON is based on the syntax of the third edition of [JavaScript](https://www.codecademy.com/resources/docs/javascript) ([ECMA-262](https://www.ecma-international.org/publications-and-standards/standards/ecma-262/)) that was released back in December 1999. However, it is language-independent.
+
+JSON is commonly used for transferring data between clients and servers for tasks such as web browsing or form submission. Companies also use JSON to allow their data to used in other applications via [API](https://www.codecademy.com/resources/docs/general/api). Some example include:
+
+- [google maps]: https://developers.google.com/maps/documentation/geocoding/start
```
Comment (yangc95) [inline content/general/concepts/json/json.md:21]:
```suggestion
- [google maps](https://developers.google.com/maps/documentation/geocoding/start)
```

File: content/general/concepts/json/json.md [hunk 2]
```diff
@@ -12,7 +12,17 @@ CatalogContent:
   - 'paths/full-stack-engineer-career-path'
 ---
 
-JavaScript Object Notation (JSON) is a data format that is readable, writable, and parsable for both humans and machines. JSON is based on the syntax of the third edition of [JavaScript](https://www.codecademy.com/resources/docs/javascript) (ECMA-262) that was released back in December 1999. However, it is language-independent.
+<link rel="canonical" href="https://www.codecademy.com/article/what-is-json" />
+
+**JavaScript Object Notation (JSON)** is a data format that is readable, writable, and parsable for both humans and machines. JSON is based on the syntax of the third edition of [JavaScript](https://www.codecademy.com/resources/docs/javascript) ([ECMA-262](https://www.ecma-international.org/publications-and-standards/standards/ecma-262/)) that was released back in December 1999. However, it is language-independent.
+
+JSON is commonly used for transferring data between clients and servers for tasks such as web browsing or form submission. Companies also use JSON to allow their data to used in other applications via [API](https://www.codecademy.com/resources/docs/general/api). Some example include:
```
Comment (yangc95) [inline content/general/concepts/json/json.md:19]:
```suggestion
JSON is commonly used for transferring data between clients and servers for tasks such as web browsing or form submission. Companies also use JSON to allow their data to use in other applications via [API](https://www.codecademy.com/resources/docs/general/api). Some examples include:
```

File: content/general/concepts/json/json.md [hunk 3]
```diff
@@ -26,7 +36,16 @@ More can be learned about JSON by visiting [json.org](https://www.json.org/).
 }
 ```
 
-As a format, JSON is textual, minimal, and universal. Multiple JSON objects can be collected in an array-like sequence:
+Data is stored in name-value pairs in object- separated by a colon `:`. The pairs themselves are separated by a comma `,`. The following [data types](https://www.codecademy.com/resources/docs/general/data-types) can be used:
```
Comment (yangc95) [inline content/general/concepts/json/json.md:39]:
```suggestion
Data is stored in an object, depicted by two curly braces `{``}`, and name-value pairs are separated by a colon `:`. The pairs themselves are separated by a comma `,`. The following are [data types](https://www.codecademy.com/resources/docs/general/data-types) that can be used:
```

File: content/general/concepts/json/json.md [hunk 4]
```diff
@@ -43,11 +62,33 @@ As a format, JSON is textual, minimal, and universal. Multiple JSON objects can
 ]
 ```
 
-An important point to note is that while it borrows from JavaScript-like syntax, JSON itself is not JavaScript.
+A few important points to note about JSON syntax:
+
+- While it is derived from from the JavaScript language, JSON itself is not JavaScript.
+- Trailing commas are forbidden
```
Comment (yangc95) [inline content/general/concepts/json/json.md:68]:
```suggestion
- Trailing commas are forbidden.
```

File: content/general/concepts/json/json.md [hunk 5]
```diff
@@ -43,11 +62,33 @@ As a format, JSON is textual, minimal, and universal. Multiple JSON objects can
...
+- Even thought JavaScript names are not this strict, JSON property names _must_ be in double-quotes.
```
Comment (yangc95) [inline content/general/concepts/json/json.md:69]:
```suggestion
- Although JavaScript names are not this strict, JSON property names must be in double quotes.
```

File: content/general/concepts/json/json.md [hunk 6]
```diff
@@ -43,11 +62,33 @@ As a format, JSON is textual, minimal, and universal. Multiple JSON objects can
...
 
 ## Bad Practices
 
 Below are a few points regarding what should be avoided when using JSON format:
 
-- It is not useful to abide by an XML-like, "attribute vs. element" framework. JSON is only about "name:value"-pairs.
+- It is not useful to abide by an XML-like, "attribute vs. element" framework. JSON is only about "name-value" pairs.
```
Comment (yangc95) [inline content/general/concepts/json/json.md:75]:
```suggestion
- It is not useful to abide by an XML-like, "attribute vs. element" framework. JSON is only about name-value pairs.
```

File: content/general/concepts/json/json.md [hunk 7]
```diff
@@ -43,11 +62,33 @@ As a format, JSON is textual, minimal, and universal. Multiple JSON objects can
...
+
+The following JSON example uses one level of nested objects, an array, and each valid data type:
+
+```pseudo
+{
+  "student": {
+    "name": "Codey",
+    "age": 30,
+    "fullTime": true,
+
```
Comment (yangc95) [inline content/general/concepts/json/json.md:88]:
```suggestion
```

**Comment by Dusch4593** [comment]
@yangc95 Implemented your feedback. 😊

**Comment by Dusch4593** [comment]
Though, I think the example could still use a nested object

**Comment by SSwiniarski** [comment]
@Dusch4593, just a note that this is updating the same file as [780](https://github.com/Codecademy/docs/pull/780), maybe we should combine the two?

**Comment by Dusch4593** [comment]
> @Dusch4593, just a note that this is updating the same file as [780](https://github.com/Codecademy/docs/pull/780), maybe we should combine the two?

@SSwiniarski Just did so! So sorry about this ☹️ I approached yesterday by a staff person and asked to modify this file because they want to include it in a piece of CC content they're writing. I should have realized you already had a PR related to this. When I merge this, I'm gonna credit you in the PR for the changes you'd already worked on. 😄

**Review by yangc95 — APPROVED** [review]

**Pull request merged**